docs/fips.md

```diff
@@ -105,19 +112,27 @@ Cilium's FIPS compliance depends on several key components:
 1. **Go Runtime**: Cilium is written in Go and relies on Go's cryptographic
 packages. A modified [Go toolchain from Microsoft] must be used.
 2. **OpenSSL**: Cilium must link against a FIPS-validated OpenSSL
-implementation. Currently, there is no FIPS base for Rocks, so it must be
-sourced, for example, from `core22/fips`.
-3. **Build Environment**: The build process must be performed on a Ubuntu Pro
-machine (details provided below).
+implementation.
+
+**NOTE**: This ROCK is bundled with a FIPS-validated OpenSSL library which is described in the 
+ROCK manifest (refer to this Discourse [post][discourse post]).
+```yaml
+...
+parts:
+  openssl:
+    plugin: nil
+    stage-packages:
+      - openssl-fips-module-3
+      - openssl
+...
+```
 
 ### Required Build Modifications
 
 To build Cilium in FIPS-compliant mode, the following requirements must be met:
 
 1. **Prerequisites**:
```

Comment:
We state the following requirements must be met and then have prerequisites which I feel is duplication